.pre-commit-config.yaml

```diff
@@ -57,3 +57,21 @@ repos:
     rev: v2.4.1
     hooks:
       - id: prettier
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.950
+    hooks:
+      - id: mypy
+        exclude: '(test_.*|noxfile)\.py'
+        args:
+          [
+            --strict,
+            --no-strict-optional,
+            --ignore-missing-imports,
+            --implicit-reexport,
+            --warn-unreachable,
+            --pretty,
+            --show-column-numbers,
+            --show-error-codes,
+            --show-error-context,
```

Comment:
Can you use a [config file](https://mypy.readthedocs.io/en/stable/config_file.html) to allow these to get picked up outside of pre-commit?